controller/proxy-injector/webhook.go

```diff
@@ -74,7 +74,11 @@ func Inject(linkerdNamespace string) webhook.Handler {
 		if ownerRef := resourceConfig.GetOwnerRef(); ownerRef != nil {
 			res, err := k8s.GetAPIResource(ownerRef.Kind)
 			if err != nil {
-				log.Warningf("skipping event for parent %s: %s", ownerRef.Kind, err)
+				if strings.Contains(err.Error(), "unimplemented resource type") {
+					log.Debugf("skipping event for parent %s: %s", ownerRef.Kind, err)
+				} else {
+					log.Warningf("skipping event for parent %s: %s", ownerRef.Kind, err)
+				}
```

Comment:
We can probably just do:

```suggestion
		log.Debugf("skipping event for parent %s: %s", ownerRef.Kind, err)
```

If you look at the code for `GetAPIResource()` we only log an error when the resource type is not recognised. There's not much point to matching the error to `unimplemented resource type` since that's always the case.

https://github.com/linkerd/linkerd2/blob/4db9c28d455fc3640fad00c46f05019867fe05e5/controller/k8s/api_resource.go#L96-L137